include/nbl/builtin/hlsl/scan/descriptors.hlsl

```diff
@@ -1,3 +1,112 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
 
+#ifndef _NBL_HLSL_SCAN_DESCRIPTORS_INCLUDED_
+#define _NBL_HLSL_SCAN_DESCRIPTORS_INCLUDED_
```

Comment:
get rid of any `descriptors` header, because we use static polymorphism in HLSL each user can provide their own accessors